<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-E4TO9P97" ignored="0" name="SATT_Selenium" tests="1" failures="1" timestamp="2022-01-06T07:59:00 ICT" time="23.095" errors="0">
  <testcase name="TC17" time="20.042" classname="Testcases.Railway.TC17">
    <failure type="java.lang.AssertionError" message="Error message is not displayed as expected. expected [There&amp;apos;re errors in the form. Please correct the errors and try again.] but found []">
      <![CDATA[java.lang.AssertionError: Error message is not displayed as expected. expected [There're errors in the form. Please correct the errors and try again.] but found []
at Testcases.Railway.TC17.TC17(TC17.java:48)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC17 -->
</testsuite> <!-- SATT_Selenium -->
